Cargo.toml

```diff
@@ -38,9 +38,10 @@ libc = "~0.2.29"
 serde_json = "~1.0.8"
 
 [features]
-mock = ["lru_time_cache/fake_clock", "safe_crypto/mock", "parsec/mock", "parsec/malice-detection"]
-mock_parsec = ["mock"]
-mock_serialise = ["mock"]
+mock_base = ["lru_time_cache/fake_clock", "safe_crypto/mock", "parsec/mock", "parsec/malice-detection"]
```

Comment:
Why is malice-detection enabled in mock, but not otherwise?

Reply:
I think that predates this PR but we just havent set that to be a default feature yet, but should be fine to set now I guess or expect parsec to have that enabled by disabled and feature gate to disable once stable.

Merging this for now though and can take this hopefully to a different PR